### DISCUSSION TITLE
Continuous Benchmarking

### DISCUSSION THREAD

**Discussion by pca006132** [elalish/manifold#1004] [General]
Thinking if we should use https://github.com/bencherdev/bencher or just write very simple script to check for relative timing. I don't think our benchmarks are complicated enough where we need to setup bencher...

And maybe we will want to use a dedicated benchmark framework like https://github.com/google/benchmark/blob/main/docs/user_guide.md to check for make sure the tests are warmup before running, with statistical checks to make sure we are not getting outliers (which can be common in some cases).

**Comment by elalish** [comment]
I trust your judgement - let's just make sure whatever we do isn't too complicated or require much maintenance.
